hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/web/resources/NamenodeWebHdfsMethods.java

```diff
@@ -1535,6 +1543,10 @@ public Void run() throws IOException {
     };
   }
 
+  private long getStateAtIndex(long[] states, int index) {
+    return states.length > index ? states[index] : -1;
+  }
```

Comment:
The same is defined in DfsClient, can we make the definition over there ``public static`` and use it here as well, rather than defining it twice?